hexhamming/_version.h

```diff
@@ -1 +1 @@
-const char _version[] = "2.2.3";
+const char _version[] = "2.2.3.1";
```

Comment:
Since this is adding to the API, let's bump at a minimum the minor version.

```suggestion
const char _version[] = "2.3.0";
```